src/ansys/grantami/recordlists/_models.py

```diff
@@ -600,6 +618,58 @@ def _to_model(self) -> models.GsaRecordListSearchCriterion:
         logger.debug(model.to_str())
         return model
 
+    def _to_2025r1_model(self) -> models2025r1.GsaRecordListSearchCriterion:
+        """
+        Generate the DTO for use with the auto-generated client code.
+
+        Compatibility version of this method for older Granta MI 2025 R1 and earlier.
```

Comment:
Comment seems a little confusing - do we mean "For older Granta MI, 2025 R1 and earlier"? If so perhaps

```suggestion
        Version of this method compatible with Granta MI 2025 R1 and earlier.
```

Reply:
I've reworked all four docstrings for the  `_to_model()` and `_to_2025r1_model()` methods to make them consistent.